specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/questionanswering-authoring.json

```diff
@@ -192,7 +192,7 @@
         }
       }
     },
-    "/query-knowledgebases/projects/{projectName}:export": {
+    "/query-knowledgebases/projects/{projectName}/:export": {
```

Comment:
According to this [issue](https://github.com/microsoft/api-guidelines/issues/265#issuecomment-920962279), the board has decided to go with a colon without a preceding directory separator.

/cc @mikekistler @JeffreyRichter @johanste

Reply:
I've started an email thread today to discuss a problem with the usage of ":" at APIM, will update all the paths as per the final recommendation.

Reply:
As discussed offline, the service will support both. @rokulka will make sure the swagger is updated prior to GA with the decision from the swagger review board.